app/src/main/res/layout/item_list_comics.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Poderia usar constraint layout aqui também =)

Não que fosse ter muito benefício nesse momento, mas deixa mais future-proof.

Reply:
Depois será alterado para Constraint Layout. Nesse início decidimos não mexer em nada de layout, só jogar os campos necessários na tela mesmo. Depois, mais pra frente, será feita uma tarefa só de layout ;)